specification/semantic_conventions.md

```diff
@@ -1,6 +1,6 @@
 # Semantic Conventions
 
-**Status**: [Stable](../README.md#versioning-and-status-of-the-specification)
+**Status**: [Mixed](../README.md#versioning-and-status-of-the-specification)
```

Comment:
We don't have a definition of "Mixed". Doesn't any non-`Stable` category here imply `Experimental`? I dunno.

Reply:
We do have a mixed: https://github.com/signalfx/gdi-specification/blob/bbc3951cabda7107a991f372a053d0f8685348ad/README.md#mixed